charts/datahub/templates/datahub-upgrade/datahub-cleanup-job-template.yml

```diff
@@ -12,6 +12,7 @@ metadata:
 spec:
   schedule: "* * * * *"
   suspend: true
+  concurrencyPolicy: {{ .Values.datahubUpgrade.cleanupJob.concurrencyPolicy | default "Allow" }}
```

Comment:
README file needs an update [here](https://github.com/acryldata/datahub-helm/blob/ae3db71b53e08731e54bf21a954dd969ceece677/charts/datahub/README.md?plain=1#L41)

Reply:
Updated the PR.